expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -187,6 +197,41 @@ void shouldReturnIndexDocumentWithTopLevelOrganizationWithoutHasPartsIfContribut
         assertThat(deepestNestedSubUnit.at(JSON_PTR_ID).textValue(), is(equalTo(affiliationToBeExpanded.toString())));
     }
 
+    @Test
+    void shouldReturnIndexDocumentWithSortedContributorsByTheirSequence()
+        throws Exception {
+        final var publication = randomBookWithManyContributors();
+        final var affiliationToBeExpanded = extractAffiliationsUris(publication).get(0);
+
+        final var mockUriRetriever = mock(UriRetriever.class);
+        mockOrganizationResponseForTopLevelAffiliation(affiliationToBeExpanded, mockUriRetriever);
+
+        var framedResultNode = fromPublication(mockUriRetriever, publication).asJsonNode();
+        var contributorsJson = framedResultNode.at("/entityDescription/contributors");
+
+        List<Contributor> contributors = objectMapper.convertValue(contributorsJson,
```

Comment:
Can it be `var`?

Reply:
Nope. Needs to be cast at some point otherwise Java think its an "Object"